dumper/lib-ext-hive-metastore/src/main/java/com/google/edwmigration/dumper/ext/hive/metastore/HiveMetastoreThriftClient_Superset.java

```diff
@@ -179,6 +179,7 @@ public String getExpandedViewText() {
       @CheckForNull
       @Override
       public String getLocation() {
+        table.getSd().getSerdeInfo().getName();
```

Comment:
Is this an accidental call to a getter which isn't assigned, or does this have a side-effect?